lib/services/platform-service.ts

```diff
@@ -431,15 +431,15 @@ export class PlatformService extends EventEmitter implements IPlatformService {
 		this.$fs.writeJson(buildInfoFile, buildInfo);
 	}
 
-	public async shouldInstall(device: Mobile.IDevice, projectData: IProjectData, outputPath?: string): Promise<boolean> {
+	public async shouldInstall(device: Mobile.IDevice, projectData: IProjectData, isRelease: IRelease, outputPath?: string): Promise<boolean> {
```

Comment:
I suggest to reuse `IBuildConfig` interface as we pass here `buildConfig` https://github.com/NativeScript/nativescript-cli/pull/3419/files#diff-513fe963f5d821a1cdb167ea91d70f1cR335

Reply:
What will changing the interface from `IRelease` to `IBuildConfig` achieve?

Reply:
We discussed and agreed to use `IRelease` interface, only `isRelease` name sounds like a boolean variable and suggest to rename it to `release: IRelease`